cypress/cypress.config.js

```diff
@@ -0,0 +1,7 @@
+const { defineConfig } = require('cypress')
```

Comment:
We already have a config file in the main branch. 
Check out `cypress/cypress.config.ts`

Reply:
Yep I'll delete it